lib/file_controller.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:finance_tracker/file_manager.dart';
+import 'package:finance_tracker/model/structure.dart';
+import 'package:flutter/material.dart';
+
+class FileController extends ChangeNotifier {
+
+  List<Transaction> listTransaction = [];
+  Transaction? structure;
```

Comment:
Trenne die datenmodelle in eigene Dateien auf und bennene die Klasse auch als Transaction. structure ist zu allegemein für diesen code

```dart
Transaction? transaction
```

bei den Tags hast du das ja auch so gemacht